test/functional/rpc_users.py

```diff
@@ -84,6 +86,39 @@ def test_auth(self, node, user, password):
         self.log.info('Wrong...')
         assert_equal(401, call_with_auth(node, user + 'wrong', password + 'wrong').status)
 
+    def test_rpccookieperms(self):
+        p = {"owner": 0o600, "group": 0o640, "all": 0o644}
+
+        if os.name == 'nt':
+            raise SkipTest(f"Skip cookie file permissions checks as OS detected as: {os.name=}")
```

Comment:
In 31cc8bc305103baf2c9979d86ef1b43dec5628cb "test: add rpccookieperms test"

This is incorrect. It causes the entire test file to be marked as skipped (different exit code, marked differently in the test runner output) even though everything else ran. The correct thing here is to skip this individual case, which can be achieved by simply returning.

Reply:
Good catch.  Might also be good to print a log message before returning for awareness.

Reply:
Oh no! fixed in 49d5bfdd7e7